.github/workflows/config-jsonschema-check.yml

```diff
@@ -108,6 +111,13 @@ jobs:
           for example in doc/examples/config/*.yaml doc/examples/config/*/*.yaml docker/example/app/config.yaml; do
             check-jsonschema --schemafile config-schema-pretty.json "$example"
           done
+      - name: Deploy JSON Schema to S3
+        if: github.ref == 'refs/heads/master' || startsWith(github.ref, 'refs/tags/')
```

Comment:
We should skip entrypoint technical tags. See [here](https://github.com/orgs/tarantool/discussions/6182#user-content-development-flow].